bot/tests/processors/test_pennychat.py

```diff
@@ -138,6 +138,16 @@ def id_mock(user_id, slack_client=None, ignore_user_not_found=True):
                         'penny_chat_description': {
                             'value': 'new_description',
                         }
+                    },
+                    'penny_chat_visibility': {
+                        'private_chat_select': {
+                            'type': 'checkboxes',
```

Comment:
Why `'type': 'checkboxes'`?

Reply:
I doubt that this code change is needed. Delete it and I bet the tests still pass because the final approach you chose doesn't even use this.